samples/excel/99-just-for-fun/path-finder-game.yaml

```diff
@@ -176,29 +173,27 @@ script:
     language: typescript
 template:
     content: |-
-        <section class="ms-Fabric ms-font-m">
+        <section>
             <p>Check whether there is a path from left to right, moving forward one cell at a time (and only straight or diagonally). </p>
         </section>
-        <h3 class="ms-font-l">Set up</h3>
-        <section class="ms-Fabric setup ms-font-m">
+        <h3>Set up</h3>
```

Comment:
The buttons in this sample no longer have styling. I think the button styling is intended to remain